int32 seed
---
float64[] initial_obs
bool success
string message